SQL/sqlRules.json

```diff
@@ -0,0 +1,42 @@
+{
+  "SQL_Server_Auth_EntraIDIntegrated": {
+    "expected": true,
+    "explanation": "The server should be integrated with Azure Active Directory for authentication and authorization. For more information: https://learn.microsoft.com/en-us/sql/relational-databases/security/authentication-access/azure-ad-authentication-sql-server-overview?view=sql-server-ver16",
+    "function": "hasAADIntegrationEnabled"
+  },
+  "SQL_Server_Auditing" : {
+    "expected": true,
+    "explanation": "The server should have auditing enabled. For more information: https://learn.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-database-engine?view=sql-server-ver16",
```

Comment:
This link seems incorrect as related to SQL Server 2022. Here is one for enabling auditing for Azure SQL: https://learn.microsoft.com/en-us/azure/azure-sql/database/auditing-setup?view=azuresql